config/src/main/java/org/springframework/security/config/annotation/authentication/configuration/EnableGlobalAuthentication.java

```diff
@@ -49,7 +49,7 @@
  * {@link AuthenticationManagerBuilder}. For example:
  *
  * <pre class="code">
- * &#064;Configuration
+ * &#064;Configuration(proxyBeanMethods = false)
```

Comment:
The spring security annotations such as `@EnableGlobalAuthentication` do not set `proxyBeanMethods=false` by default, to avoid breaking changes.
I have updated the Javadoc to encourage users to add the additional annotation `@Configuration(proxyBeanMethods = false)`.